restli-int-test-server/src/main/java/com/linkedin/restli/examples/greetings/server/defaults/FooDefaultFillResources.java

```diff
@@ -0,0 +1,48 @@
+package com.linkedin.restli.examples.greetings.server.defaults;
+
+import com.linkedin.restli.examples.defaults.api.FooDefault;
+import com.linkedin.restli.server.CollectionResult;
+import com.linkedin.restli.server.annotations.Finder;
+import com.linkedin.restli.server.annotations.QueryParam;
+import com.linkedin.restli.server.annotations.RestLiCollection;
+import com.linkedin.restli.server.annotations.RestMethod;
+import com.linkedin.restli.server.resources.CollectionResourceTemplate;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@RestLiCollection(name = "fooDefaultFilling", namespace = "com.linkedin.restli.examples.defaults.api")
+public class FooDefaultFillResources extends CollectionResourceTemplate<Integer, FooDefault>
```

Comment:
1. Similar to Karthik's other comment, this resource should be named according to what it's actually doing. Something like `FillInDefaultsCollectionMetadataResource` ("Resource" should not be pluralized).
2. Also consider whether or not we need this extra resource. Keep in mind that another resource `FieldFillInDefaultResources` exists and it appears to have similar functionality. Can we reuse that?

Reply:
Yes, end up using existing resources